### ISSUE TITLE
Updated Dunning guides

### ISSUE THREAD

**Issue by bkobash** [bkobash/ios-designers-swift#2]
I just hooked these together with modal views. I suspect that, in reality, the whole waypoint interface (i.e., the three cards in the demo) would appear in a single view controller, so the user can scroll through them easily. @codepathreview

![dunning-guides](https://cloud.githubusercontent.com/assets/1093474/5928817/de478810-a632-11e4-8bec-913ea341eb22.gif)

**Comment by timothy1ee** [comment]
Yep, do you remember when we used the UIPageViewController? That's how I'd probably implement it for real. You can also use a UIScrollView if you're just prototyping.
